packages/block-library/src/post-author/index.php

```diff
@@ -130,7 +126,7 @@ function render_block_core_post_author( $attributes, $content, $block ) {
 		: '';
 
 	return sprintf( '<div %1$s %2$s>', $class_attribute, $style_attribute ) .
-		( $attributes['showAvatar'] ? '<div class="wp-block-post-author__avatar">' . $avatar . '</div>' : '' ) .
+		( ! empty( $attributes['showAvatar'] ) ? '<div class="wp-block-post-author__avatar">' . $avatar . '</div>' : '' ) .
 		'<div class="wp-block-post-author__content">' .
 			( ! empty( $byline ) ? '<p class="wp-block-post-author__byline">' . $byline . '</p>' : '' ) .
 			'<p class="wp-block-post-author__name">' . get_the_author_meta( 'display_name', $author_id ) . '</p>' .
```

Comment:
not related, but this looks cryptic and stressful on eyes!

Reply:
Agreed, the ternary operators inside a multi-line string concatenation isn't particularly readable and probably leads to more to bugs. 

Not a bad idea to refactor this at some point to make it a bit more readable.